controllers/__init__.py

```diff
@@ -92,7 +92,7 @@ def upsert_evc(self, evc: Dict) -> Optional[Dict]:
         updated = self.db.evcs.find_one_and_update(
             {"_id": evc["id"]},
             {
-                "$set": model.dict(exclude={"inserted_at"}, exclude_none=True),
+                "$set": model.dict(exclude={"inserted_at"}, exclude_unset=True),
```

Comment:
Looks like it was at the `exclude_none=True` that we sent to the DB. This means that the problem to set null to `queue_id` was not only from the UI but the API request PATCH also had this problem. Changing it to `exclude_unset=True` will make the EVC to have extra null values `queue_id, sb_priority, end_date, owner`. This is `upsert_evc` receives `self.as_dict()` from the EVC which has everything initialize with a value or None. Is this okay? The other option I see is to set `queue_id` as numeric only so it is easier to manage.

Reply:
@Alopalao `exclude_unset` looks like a good intermediary compromise in this case. 

> We could have some vulnerabilities if upsert gets a field with a false None because the EVC content was not retrieved from database.

It might be worth to take the opportunity to update EVCBaseDoc `metadata: Optional[dict] = None` as a `metadata: dict = {}` instead of `None`, `None` here wouldn't be ideal since we know for a fact that `None` metadata has no valid semantics, since by default it should be a dict. Anything else that indeed has proper `None` semantics like `queue_id, sb_priority, end_date, owner, flow_removed_at` then it's OK.

Currently, mef_eline will load first from the DB, yes we don't want to have that vulnerability, but if the model is allowing an Optional, and `None` has a semantics it's a fair game, but let's ensure that loading will always load as expected. Lastly, it might be worth to run mef e2e tests, but yes, I wouldn't expect it to break anything

Reply:
After last commit [409ff3008b7aa0afd20df59c207c0f7008cfc2f4](https://github.com/kytos-ng/mef_eline/pull/348/commits/409ff3008b7aa0afd20df59c207c0f7008cfc2f4)

E2E results:
```
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-7.2.0, pluggy-1.2.0
rootdir: /tests
plugins: timeout-2.1.0, rerunfailures-10.2, anyio-3.6.2
collected 239 items

tests/test_e2e_01_kytos_startup.py ..                                    [  0%]
tests/test_e2e_05_topology.py ..................                         [  8%]
tests/test_e2e_10_mef_eline.py ..........ss.....x.....x................  [ 25%]
tests/test_e2e_11_mef_eline.py ......                                    [ 27%]
tests/test_e2e_12_mef_eline.py .....Xx.                                  [ 30%]
tests/test_e2e_13_mef_eline.py .....xs.s......xs.s.XXxX.xxxx..X......... [ 48%]
...                                                                      [ 49%]
tests/test_e2e_14_mef_eline.py x                                         [ 49%]
tests/test_e2e_15_mef_eline.py ..                                        [ 50%]
tests/test_e2e_20_flow_manager.py .....................                  [ 59%]
tests/test_e2e_21_flow_manager.py ...                                    [ 60%]
tests/test_e2e_22_flow_manager.py ...............                        [ 66%]
tests/test_e2e_23_flow_manager.py ..............                         [ 72%]
tests/test_e2e_30_of_lldp.py ....                                        [ 74%]
tests/test_e2e_31_of_lldp.py ...                                         [ 75%]
tests/test_e2e_32_of_lldp.py ...                                         [ 76%]
tests/test_e2e_40_sdntrace.py ...........                                [ 81%]
tests/test_e2e_41_kytos_auth.py ........                                 [ 84%]
tests/test_e2e_50_maintenance.py ........................                [ 94%]
tests/test_e2e_60_of_multi_table.py .....                                [ 97%]
tests/test_e2e_70_kytos_stats.py RRFRRFRRFRRFRRFRRFR                     [100%]R [100%]F [100%]
```

Forgot to update `kytos-docker`
```
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-7.2.0, pluggy-1.2.0
rootdir: /tests
plugins: timeout-2.1.0, rerunfailures-10.2, anyio-3.6.2
collected 7 items

tests/test_e2e_70_kytos_stats.py ....R...                                [100%]
```